auth-web/src/components/auth/account-settings/account-info/AccountInfo.vue

```diff
@@ -9,15 +9,6 @@
           {{ errorMessage }}
         </v-alert>
 
-        <div v-can:VIEW_ACCOUNT.hide>
-          <div class="nv-list-item mb-10">
-            <div class="name" id="accountName">Account Name</div>
-            <div class="value" aria-labelledby="accountName">
-              <div class="value__title">{{ currentOrganization.name }}</div>
-            </div>
-          </div>
-        </div>
-
```

Comment:
this code block was for showing name to USER/COORDINATOR who couldn't change the name...now name is disabled for everyone..so no need of it I think